scripts/migration/migrate-istio

```diff
@@ -0,0 +1,1360 @@
+#!/usr/bin/env bash
+set -CeEu
+set -o pipefail
+
+SCRIPT_NAME="${0##*/}"
+
+PROJECT_ID=""
+FLEET_PROJECT_ID=""
+CLUSTER_NAME=""
+CLUSTER_LOCATION=""
+
+DRY_RUN=0
+VERBOSE=0
+ENABLE_DEPENDENCIES=0
+UPGRADE_GATEWAYS=1
+
+CONFIGURED=0
+MIGRATE=0
+PRE_MIGRATE=0
+INSTALL_ASM=0
+RELABEL=0
+FINALIZE=0
+ROLLBACK=0
+
+ASM_VERSION="1.10.2-asm.2"
+KPT_URL="https://github.com/GoogleCloudPlatform/anthos-service-mesh-packages"
+KPT_URL="${KPT_URL}.git/asm@release-1.10-asm"; readonly KPT_URL
+REVISION_LABEL="asm-1102-2"; readonly REVISION_LABEL
```

Comment:
nit: any intention of allowing users to configure this in future? What if users want to jump to ASM 1.11?

Reply:
I'd rather give us the opportunity to test it internally with a new version before allowing anyone to set the version at runtime.